distro.py

```diff
@@ -0,0 +1,131 @@
+"""
+@author: Mark V Systems Limited
+(c) Copyright 2011 Mark V Systems Limited, All rights reserved.
```

Comment:
nit: saw your separate issue on changing the copyright. Should this be Workiva?

Reply:
It should. We're planning to update all of these in one fell swoop. I didn't want to touch it here (and possibly have to hold this while the copyright description was approved).